force-app/main/default/translations/ca.translation-meta.xml

```diff
@@ -1698,8 +1922,16 @@
         <label>No podem desar els horaris de l’oferta de cursos perquè l&apos;hora d&apos;inici no coincideix amb l’hora d&apos;inici del bloc de temps associat. Editeu l&apos;hora d&apos;inici, especifiqueu un bloc de temps diferent o bé no especifiqueu cap bloc de temps.</label>
         <name>timeBlockStartTimeMismatch</name>
     </customLabels>
+    <customTabs>
+        <label><!-- Education Cloud Settings --></label>
+        <name>Education_Cloud_Settings</name>
+    </customTabs>
     <customTabs>
         <label>Configuració d&apos;EDA</label>
         <name>HEDA_Settings</name>
     </customTabs>
+    <customTabs>
+        <label><!-- Settings Health Check --></label>
+        <name>Settings_Health_Check</name>
+    </customTabs>
```

Comment:
This also needs to be removed in every file since it is unmanaged.
```suggestion
    <customTabs>
        <label><!-- Settings Health Check --></label>
        <name>Settings_Health_Check</name>
    </customTabs>
```